modules/local/get_query_groups.nf

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env nextflow
+
+nextflow.preview.dsl=2
+
+params.query = '/Users/tomaszwrzesinski/Documents/nextflow-hackathon-2020/sim-seqer/tests/data/heavy_tiny/query.csv'
+params.match = 'h_v h_cdr3_len'
+params.align = 'h_pep_mature'
+params.list_of_reference_files = ['root/h_cdr3_len.h_v/12.IGHV1-69/ref.csv', 'root/h_cdr3_len.h_v/17.IGHV1-46/ref.csv', 'root/h_cdr3_len.h_v/12.IGHV1-46/ref.csv']
+
+process GET_QUERY_GROUPS {
+    publishDir 'results/', mode: 'copy'
+
+
+    input:
+    path query
+    val list_of_reference_files
+    val align
+    val match
+
+    output:
+    path 'fasta_dir/*/*.fasta', emit: fasta_dir
+    path 'query_fasta.csv', emit: query_fasta_csv
+    path 'ref_paths.csv', emit: ref_paths_csv
+
+
+    script:
+    list_of_reference_files_space_separated = list_of_reference_files.join(' ')
```

Comment:
I'm not sure if this one is needed. I've never used this conversion usually .collect() on channel is enough to have correct nargs+ :D

Reply:
Not really, be aware that we need a space-separated string, not a list (which is an output of `.collect()` method).